dts/arm/st/h7/stm32h723.dtsi

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2020 Alexander Kozhinov
+ * AlexanderKozhinov@yandex.com
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <st/h7/stm32h7.dtsi>
+
+/ {
+	/* DTCM memory directly coppled to CPU */
+	dtcm: memory@20000000 {
+		compatible = "arm,dtcm";
+		reg = <0x20000000 DT_SIZE_K(128)>;
+	};
+
+	/* AXI SRAM in D1 domain (AXI bus) */
+	axi_sram: memory@24000000 {
```

Comment:
In other boards this is usually named `sram0` (the actual memory used by Zephyr).  E.g. https://github.com/zephyrproject-rtos/zephyr/blob/master/dts/arm/st/h7/stm32h743.dtsi#L11

Reply:
I think we need to find a way to define name conventions for ST's SRAM region names (please take a look @erwango). Otherwise I will need to change it another multiple times.

Reply:
Chanegd to
 - sram0: memory@24000000